utils/nctl/sh/scenarios/event_stream.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/utils/main.sh
+source "$NCTL"/sh/views/utils.sh
+source "$NCTL"/sh/node/svc_"$NCTL_DAEMON_TYPE".sh
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+# Exit if any of the commands fail.
+set -e
+
+TMP_EVENTS_FILE=tmp_events_main.txt
+
+#######################################
+# Runs an integration tests that verifies if events are routed via correct SSE endpoints.
+# Prior to https://github.com/casper-network/casper-node/issues/4314 there were 3 different
+# endpoints, serving different evvents. The 3 events that this test verifies used to be
```

Comment:
nit: typo `events` instead of `evvents`

Reply:
Fixed in d32e1e44754aaa86d750d3a5f84bcc6a29be8c9f